certified-connectors/Aranda Service Management/apiProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "hostUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "clearText": true,
+            "tabIndex": 0
+          },
+          "description": "Host ASMS",
+          "displayName": "Host ASMS",
+          "tooltip": "URL host"
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "clearText": false,
+            "tabIndex": 1
+          },
+          "description": "Authorization token",
+          "displayName": "Authorization token",
+          "tooltip": "Authorization token"
+        }
+      }
+    },
+    "iconBrandColor": "#f3836f",
+    "capabilities": [],
+    "policyTemplateInstances": [
```

Comment:
Can you please fix below validation error? Please refer this schema. 

@{level=Error; message=JSON does not match any schemas from 'anyOf'.; valuePath=line: 54 position: 7 path: '[0].properties.policyTemplateInstances[2]'; value=}

Reply:
unnecessary property was eliminated